### DISCUSSION TITLE
.lib files not available on windows

### DISCUSSION THREAD

**Discussion by RoshiniAithal** [ImageMagick/ImageMagick#7293] [Development]
### ImageMagick version

7.1.1.31

### Operating system, version and so on

Windows 10

### Description

I downloaded ImageMagick 7.1.1.31 . only dlls are installed, no lib files are available.
when i include API s of ImageMagick, getting following errors
Error 76 error LNK2019: unresolved external symbol "__declspec(dllimport) void __cdecl Magick::InitializeMagick(char const *)" (_imp?InitializeMagick@Magick@@YAXPBD@Z) referenced in function

Please let me know how to get lib files

**Comment by dlemstra** [comment]
The `.lib` files are no longer available you will need to link to the exports of the `.dll` file instead. And we have already answered questions about this. Next time please make sure you search our issues and discussions.
